app/models/metadata.rb

```diff
@@ -64,7 +68,14 @@ def self.sanitize_params(filters)
         query[filter['name']] = filter['options']
       end
     end
-    query.delete_if { |_k, v| v.empty? }
+    query.delete_if { |k, v| v.empty? || 'factsheet'.include?(k) }
```

Comment:
I'm not 100% sure about this. If you would need to remove options from the `query` and one is `factsheet` why do you need to consider this differently?
Also, couldn't that be simply `'factsheet' == k`?